TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -4,14 +4,21 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
-    <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
-    <OutputPath>bin\$(Configuration)\netcoreapp1.0</OutputPath>
+    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
+    <OutputPath>bin\$(Configuration)</OutputPath>
```

Comment:
I think this can be removed completely.  No reason to set it here, as the sdk .targets does it for us.